### PR TITLE
Updating Contribution Content

### PR DESCRIPTION
**Pull request by at669**
Re: [AIRO-486](https://jira.unity3d.com/browse/AIRO-486)

- Adding code of conduct
- Adding contribution guidelines
- Adding readme info on contributing, community, support
- Updating license appendix
- Adding bug report & feature request issue templates

Will finalize this PR before applying the finalized versions to the other repositories.

Remaining TODO: CLA

### CODE REVIEW

File: CONTRIBUTING.md
Comment (mrpropellers) [inline CONTRIBUTING.md:41]: I'm going to be looking into whether or not it's possible to run CI *before* merges in the case where we get a PR from someone not on the code owner's list.  In cases where we can't follow up easily with the submitter if the code breaks `dev` after merge, we really want to be confident that the tests are going to pass before it goes in.

File: CONTRIBUTING.md
Comment (mrpropellers) [inline CONTRIBUTING.md:47]: Might want to point out here that we expect submissions to follow these conventions even if the code they are changing does not, and also they should be aware of the [Unity coding package](https://github.com/Unity-Technologies/com.unity.coding/blob/master/UnityCoding/Packages/com.unity.coding/README.md) since we have a couple special conventions that will supersede the standard C# ones.

File: CONTRIBUTING.md
Comment (mrpropellers) [inline CONTRIBUTING.md:56]: Do we have one of these?  If not, do we need one?  Probably not, unless there's a company mandate that says otherwise.

**Review by mrpropellers — CHANGES_REQUESTED** [review]
Thanks for pushing on this long-overdue work!  Left a few comments, and I'm to do some investigation into the granularity of Yamato's trigger filters to get a better sense of what our CI rules should be.

File: CONTRIBUTING.md
Comment (at669) [reply CONTRIBUTING.md:56]: I've started the process for setting it up on this repo, but I was planning on holding off until this PR was finalized and merged--I believe it's required per the [License instructions](https://docs.google.com/document/d/19asl21ovg6SKY9k5xnukFAXCYHx7u89MQOoWEtrMZEU/edit#heading=h.auhk2j9xy48i).

File: CONTRIBUTING.md
Comment (at669) [reply CONTRIBUTING.md:47]: Will add the note about following conventions even if the code they're changing does not! Re: coding package, I think that package is private though, and I haven't been able to find a public-facing version of Unity standards.

File: CONTRIBUTING.md
Comment (mrpropellers) [reply CONTRIBUTING.md:56]: Hmm yeah, you're right... well, the [Unity Coding package](https://docs.unity3d.com/Packages/com.unity.coding@0.1/manual/index.html) itself is public;  that might be the best resource we can provide.

File: CONTRIBUTING.md
Comment (mrpropellers) [reply CONTRIBUTING.md:47]: Hmm yeah, you're right... well, the [Unity Coding package](https://docs.unity3d.com/Packages/com.unity.coding@0.1/manual/index.html) itself is public;  that might be the best resource we can provide.

File: CONTRIBUTING.md
Comment (mrpropellers) [reply CONTRIBUTING.md:41]: Looks like there's no way to change test behavior based on the actual submitter. I think the best we can do is change it to run CI on every PR submission and give code owners the right to over-ride the status check on as as-needed basis.

**Review by mrpropellers — APPROVED** [review]

**Pull request merged**
